src/test/java/pages/LoginPage.java

```diff
@@ -0,0 +1,31 @@
+package pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+public class LoginPage extends BasePage {
+
+
+    public LoginPage(WebDriver givenDriver) {
+        super(givenDriver);
+    }
+
+
+    public static void provideEmail(String email) {
+        WebElement emailField = driver.findElement(By.cssSelector("[type = 'email']"));
+        emailField.sendKeys(email);
+    }
+
+    public static void providePassword(String password) {
+        WebElement passwordField = driver.findElement(By.cssSelector("[type = 'password']"));
+        passwordField.sendKeys(password);
+    }
+
+    public static void clickSubmit() {
+        WebElement submitButton = driver.findElement(By.cssSelector("button[type = 'submit']"));
+        submitButton.click();
+    }
```

Comment:
We can make the xpath or css the **class variable (Locator)**

![image](https://user-images.githubusercontent.com/104761482/224471053-997d5f35-fec3-4d53-bf2e-b8ba12103af5.png)